### ISSUE TITLE
[Machine Learning] Getting started with Explainable AI using ELI5

### ISSUE THREAD

**Issue by francis966** [section-engineering-education/engineering-education#4358]
#### Proposed title of article
[Machine Learning] Getting started with Explainable AI using ELI5 

#### Proposed article introduction
Explainable AI in machine learning is the process of understanding how a machine learning model achieved a certain solution y following its decision-making process. Explainable AI finds out why a model makes certain decisions during the prediction phase. This ensures we have a fair, accountable, and transparent model.

Explainable AI solves the black box problem by explaining the hidden functionalities of the model and explains how a model reached a certain conclusion.
In Explainable AI we try to answer the question; why should we trust the model? and why did the model make this conclusion?
Explainable AI has the following benefits.

- It detects errors and bugs in the model.
- It detects bias during model training.
- To know model reliability.
- To improve model performance and generalization.
- To identify wrong classifications during predictions.

In this tutorial, we shall use ELI5 in building our model. ELI5 is an acronym for ‘Explain Like I’m 5’. It’s a Python library used to inspect ML classifiers and explain their predictions results.

#### Article quality
In this tutorial, we will build a model from the initial stages and follow all the machine learning stages until we come up with a custom model. 

After building the model, we then use this ELI5 to explain the predictions made by the model. This enables us to trust the model so that we can not just the model results blindly. This is important in models such as health systems, self-driving cars in which an error can cause a great loss.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by WanjaMIKE** [comment]
@francis966 you suggested the same topic as @jamesomina99 -> https://github.com/section-engineering-education/engineering-education/issues/4324 What if the student decides to modify it and reopen later on? I am closing this to avoid such conflicts.
